src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,81 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  searchMovie: (title: string) => void;
+  isMovieFound: boolean;
+  foundedMovie: Movie | null;
+  addToList: () => void;
+};
+
+export const FindMovie: React.FC<Props> = ({
+  searchMovie,
+  isMovieFound,
+  foundedMovie,
+  addToList,
+}) => {
+  const [searchQuery, setSearch] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearch(event.target.value);
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
```

Comment:
It is red all the time, the user will think that he made some type of mistake, fix it
![image](https://user-images.githubusercontent.com/37161315/149943519-a1fc3e66-1302-4395-8d12-1ffd0c43ecd3.png)